lib/facter.rb.in

```diff
@@ -11,7 +11,7 @@ module CFacter
   # @api public
   def self.initialize
     begin
-      require "#{ENV['CFACTER_DIR'] || '${CMAKE_INSTALL_PREFIX}'}/${LIBFACTER_INSTALL_DESTINATION}/libfacter.so"
+      require "#{ENV['FACTER_DIR'] || '${CMAKE_INSTALL_PREFIX}'}/${LIBFACTER_INSTALL_DESTINATION}/libfacter.so"
```

Comment:
If we want to behave as `facter.rb`, we shouldn't have an `initialize` method, but instead initialize libfacter automatically.  Thus, users that are doing `require 'facter'` get the expected API without having to do `Facter.initialize`.